### DISCUSSION TITLE
Exclude certain tags/releases from `releases.atom`

### DISCUSSION THREAD

**Discussion by dsotirakis** [community/community#35990] [Repositories]
Hello everyone! 👋 
We are trying to test our release pipelines in our OSS repo, by tagging and pushing a dummy `v0.0.0-test` version of our product. 
We've got the feedback that this version appears on people's RSS feed (they have been subscribed to `releases.atom` and it appears there.
I was wondering if there's a way to prevent tags from appearing in the `releases.atom` file, or do something different (like a `--dry-run` flag) when we push the tag to the remote.
Thanks for your help!

**Comment by dsotirakis** [comment]
Or a simpler question would be if people who are subscribed to github RSS feed can subscribe **only** to releases and not tags?

**Reply by dgw** [reply]
The existence of separate `releases.atom` and `tags.atom` feeds implies that the feeds' contents reflect what's shown on the respective tabs. That is, if a Release is created, it's shown on the Releases tab and included in the `releases.atom` feed, but a tag that isn't published as a release appears only in the Tags tab and `tags.atom`.

I intentionally pushed only a tag for a compatibility release (changing package name), leaving the actual Release for the "real" new version, but users following my repo said the tag also appeared in their release feeds—which I verified myself.

To me, this seems like a bug.
